sensor_msgs/Image mask
---
# response params
bool success
ContactGrasp[] grasps